### PR TITLE
Connection sort change

### PR DESCRIPTION
**Pull request by davidkelly**
Now, offline people will sort below online people (when sorted in ascending order).

### CODE REVIEW

**Comment by davidkelly** [comment]
Test Plan

You need a few connections, only some of whom are online.  
Open up the PAL
go to the Connections tab
sort on location (click the location header)
online people should come up first, alphabetically by location in ascending order (a-z), with offline people (blank locations) at the bottom
sorting the other way (click the header again) puts them on top, followed by online people sorted in descending order (z-a)

**Comment by hifi-gustavo** [comment]
Build console output for commit: 9376ac646f71fb4454aa3f64c034cc36cf0c2dcd available at: https://gist.github.com/273c633a0a0e0d6f94ecb15fd0b803b3<br/><br/>Installers available for download at: <br/>Windows: http://cachefly.highfidelity.com/HighFidelity-Beta-PR10202-9376ac6.exe<br/>OS X: http://cachefly.highfidelity.com/HighFidelity-Beta-PR10202-9376ac6.dmg

File: interface/resources/qml/hifi/Pal.qml [hunk 1]
```diff
@@ -1401,6 +1401,13 @@ Rectangle {
         var selectedIDs = getSelectedConnectionsUserNames();
         connectionsUserModelData.sort(function (a, b) {
             var aValue = a[sortProperty].toString().toLowerCase(), bValue = b[sortProperty].toString().toLowerCase();
+            if (!aValue && !bValue) {
+                return 0;
+            } else if (!aValue) {
+                return after;
+            } else if (!bValue) {
+                return before
```
Comment (zfox23) [inline interface/resources/qml/hifi/Pal.qml]: Missing semicolon :)

**Comment by OmegaHeron** [comment]
Tested Windows 10
QA Pass - compared to Master this now sorts as one would expect for given field.

**Comment by hifi-gustavo** [comment]
Build console output for commit: 058a5b14a42c14e8038e5dc3e38f00df93b347af available at: https://gist.github.com/273c633a0a0e0d6f94ecb15fd0b803b3<br/><br/>Installers available for download at: <br/>Windows: http://cachefly.highfidelity.com/HighFidelity-Beta-PR10202-058a5b1.exe<br/>OS X: http://cachefly.highfidelity.com/HighFidelity-Beta-PR10202-058a5b1.dmg

**Pull request merged**